tests/config_test.py

```
@@ -204,3 +205,230 @@ def load(f):
+    def test_load_config_invalid_toml(self):
+        """Test handling of invalid TOML syntax."""
+        invalid_toml = b"""
+[incomplete
+missing closing bracket
+"""
+        with tempfile.NamedTemporaryFile(mode="wb", suffix=".toml", delete=False) as f:
+            f.write(invalid_toml)
+            f.flush()
+
+            try:
+                with pytest.raises(Exception):  # Should raise a TOML parsing error
```

Ruff (0.14.14)

[warning] Do not assert blind exception: `Exception`

(B017)